packages/experience-builder-sdk/package.json

```diff
@@ -9,9 +9,9 @@
   "license": "MIT",
   "scripts": {
     "dev": "vite",
-    "prepare": "node ./bin/injectSDKVersion.cjs",
-    "prebuild": "node ./bin/injectSDKVersion.cjs",
-    "build": "vite build",
+    "prepare": "node ./bin/injectSDKVersion.cjs && relative-deps",
+    "prebuild": "rimraf dist && node ./bin/injectSDKVersion.cjs",
+    "build": "rollup -c rollup.config.mjs",
     "watch": "vite build --watch",
```

Comment:
is there a chance we could entirely cut `vite` out?

It see that there is still this script, the plugins and the config file in the project